test/e2e/metrics_e2e_test.go

```diff
@@ -170,17 +170,53 @@ var _ = Describe("Metrics are generated for OLM managed resources", func() {
 				}).Should(And(
 					Not(ContainElement(LikeMetric(
 						WithFamily("subscription_sync_total"),
-						WithLabel("name", "metric-subscription-for-update"),
-						WithLabel("channel", stableChannel),
+						WithName("metric-subscription-for-update"),
+						WithChannel(stableChannel),
```

Comment:
Should this matcher (and the new ones added in this PR) also specify the package?